RELEASE/scripts/autoscend.ash

```diff
@@ -10560,33 +10560,37 @@ boolean LX_craftAcquireItems()
 		}
 	}
 
-	if((item_amount($item[snow berries]) == 3) && (my_daycount() == 1) && get_property("auto_grimstoneFancyOilPainting").to_boolean())
+	// Snow Berries can be acquired out of standard by using Van Keys from NEP. We need to check to make sure they are usable.
+	if(is_unrestricted($item[snow berries]))
```

Comment:
generally use `auto_is_valid` instead as it checks other things as well (like g lover and bees hate you usability)

Reply:
Fixed and merging! thanks!